sql_test.go

```diff
@@ -10,10 +10,10 @@ import (
 )
 
 func TestScan(t *testing.T) {
-	var stringTest string = "f47ac10b-58cc-0372-8567-0e02b2c3d479"
```

Comment:
If this is going to change I would switch to:

var (
    byteTest = ...
    stringTest = ...
    badTypeTest = ...
    invalidTest = ...
)